README.md

```diff
@@ -196,8 +196,6 @@ configuration:
 Shell aliases and scripts:
 
 * `b` for `bundle`.
-* `g` with no arguments is `git status` and with arguments acts like `git`.
```

Comment:
We still have this as a function: https://github.com/thoughtbot/dotfiles/blob/master/zsh/functions/g